modules/ozoneBidAdapter.js

```diff
@@ -355,13 +365,34 @@ export const spec = {
     if (config.getConfig('coppa') === true) {
       deepSetValue(ozoneRequest, 'regs.coppa', 1);
     }
+    let ozUuid = generateUUID();
+    if (this.isBatchRequests()) {
+      logInfo('going to batch the requests');
+      let arrRet = []; // return an array of objects containing data describing max 10 bids
+      for (let i = 0; i < tosendtags.length; i += 10) {
+        ozoneRequest.id = ozUuid; // Unique ID of the bid request, provided by the exchange. (REQUIRED)
+        ozoneRequest.imp = tosendtags.slice(i, i + 10);
+        ozoneRequest.ext = extObj;
+        deepSetValue(ozoneRequest, 'source.tid', ozUuid);// RTB 2.5 : tid is Transaction ID that must be common across all participants in this bid request (e.g., potentially multiple exchanges).
```

Comment:
you shouldn't be setting source.tid as any value other than auction id

Reply:
have removed - thanks - please check latest commit